lib/patientFinder/utils.rb

```diff
@@ -0,0 +1,165 @@
+require 'rubygems'
+require 'nokogiri'
+require 'health-data-standards'
+require 'zip/zipfilesystem'
+require 'fileutils'
+require 'health-data-standards'
+require 'pry'
+Mongoid.load!("mongoid.yml", :development)
+
+module PatientFinder
+# a collection of utilities grouped together under the same roof
+    class Utils
+        
+        #some of records are created ambiguous on purpose (fr example some of them contain only encounters) - in order to disambiguate human intervention is needed - the following hash resolves the ambiguos cases
+        AMBIGUOUS_PATIENTS = {["ee85c89f24946e2ddca12c6edc5181dc", "eb6af018e11320f5c163624beb83767d"] => "ee85c89f24946e2ddca12c6edc5181dc"}            
```

Comment:
I know  - I initially have it in one single block between {} - after that I changed to multiple line declaration to make it easier to read - but if the general convention is to have everything between {} I am conforming to it. Nathan mention below are good - unfortunately I couldn't replace {} w/o getting syntax errors - do you have any example for hash build using do...end